src/Nethermind/Nethermind.Shutter/ShutterP2P.cs

```diff
@@ -42,12 +42,13 @@ public ShutterP2P(IShutterConfig shutterConfig, ILogManager logManager, IFileSys
     {
         _logger = logManager.GetClassLogger();
         _cfg = shutterConfig;
+        DisconnectionLogTimeout = TimeSpan.FromMilliseconds(_cfg.DisconnectionLogTimeout);
```

Comment:
This will also have the effect of the disconnect message only being displayed every 20 min or whatever the setting is.